### PR TITLE
Email Copy Audit

### PR DESCRIPTION
**Pull request by frankchen07**
##### Description

Grammar, punctuation, and spelling changes for the bulk of Gitcoin email copy. Thanks to @octavioamu for getting me set up.

##### Refers/Fixes

N/A

##### Testing

N/A

### CODE REVIEW

File: app/retail/templates/emails/kudos_mkt.html [hunk 1]
```diff
@@ -80,8 +80,8 @@ <h2>{% trans "Kudos can be used to" %}</h2>
 <p  style="line-height: 2; font-size:18px;">
   {% blocktrans %}
   Recognize Gitcoiners you’ve worked with <br>
-  Understand other Gitcoiner skillsets<br>
-  See who Gitcoiners have worked iwth<br>
+  Understand other Gitcoiner's skillsets<br>
```
Comment (danlipert) [inline app/retail/templates/emails/kudos_mkt.html]: Should this be `Gitcoiners'`? I'm not a grammar expert though....

File: app/retail/templates/emails/bounty/no_applicant_reminder.html [hunk 2]
```diff
@@ -77,11 +77,11 @@
 </style>
 
 <div class="bounty-box">
-  <h1>Get more applicants on your bounty</h2>
+  <h1>Get more applicants on your bounty!</h2>
 
   <p class="mb-0 mt-4">We noticed you dont have any contributors on your bounty.</p>
   <p class="mt-1">
-    Go to the Users Directory to discover and invite the best contributors
+    Go to the User Directory to discover and invite the best contributors
```
Comment (thelostone-mc) [inline app/retail/templates/emails/bounty/no_applicant_reminder.html]: would we wanna wrap this within a link @frankchen07  ?

```html
<p class="mt-1">
    Go to the 
    <a  href="https://gitcoin.co/users/" target="_blank">User Directory </a>
    to discover and invite the best contributors for your bounty!
</p>```

Comment (octavioamu) [reply app/retail/templates/emails/bounty/no_applicant_reminder.html]: I think is not necessary since the button with the link is behind this

Comment (frankchen07) [reply app/retail/templates/emails/bounty/no_applicant_reminder.html]: agree, there's a button with the directory link in later lines (113-114)

File: app/retail/templates/emails/kudos_mkt.html (hunk 1 above)
Comment (frankchen07) [reply app/retail/templates/emails/kudos_mkt.html]: sorry, finally got to this. yes you are correct, will push this small change.

**Pull request merged**